lib/manageiq/appliance_console/database_configuration.rb

```diff
@@ -275,7 +275,7 @@ def validate_encryption_key!
 
     def do_save(settings)
       require 'yaml'
-      File.write(DB_YML, YAML.dump(settings))
+      File.write(DB_YML, YAML.dump(settings), :perm => 0o440)
```

Comment:
Should this be 660?

Reply:
After looking into it a bit more, I think 440 is okay, but we do have other writers of this file that probably need to be updated:
- https://github.com/ManageIQ/manageiq/blob/master/lib/manageiq/environment.rb#L43
- https://github.com/ManageIQ/manageiq-postgres_ha_admin/blob/master/lib/manageiq/postgres_ha_admin/config_handler/rails_config_handler.rb#L31
- fix auth

Reply:
feels like we do this stuff in quite a few places.

I was adding the chmod code into our rpm process as well.

v2 key also is handles in fix auth, here, and chmodded in the rpm.
We have other areas that copy the sample files.

Thank you for the links. I'll look into changing all of these or if possible, find a way to reduce all these